packages/opentelemetry-web/src/WebTracer.ts

```diff
@@ -14,25 +14,43 @@
  * limitations under the License.
  */
 
+import { BasePlugin } from '@opentelemetry/core';
 import { BasicTracer, BasicTracerConfig } from '@opentelemetry/tracing';
 import { StackScopeManager } from './StackScopeManager';
 
+/**
+ * WebTracerConfig provides an interface for configuring a Web Tracer.
+ */
+export interface WebTracerConfig extends BasicTracerConfig {
+  /**
+   * plugins to be used with tracer, they will be enabled automatically
+   */
+  plugins?: BasePlugin<unknown>[];
+}
+
 /**
  * This class represents a web tracer with {@link StackScopeManager}
  */
 export class WebTracer extends BasicTracer {
   /**
    * Constructs a new Tracer instance.
+   * @param {WebTracerConfig} config Web Tracer config
```

Comment:
Nit: what would you think about removing the type annotation here in the JSDoc since it's already present in TypeScript types and so feels duplicated in the JSDoc?